tests/app/tests/ns-location-strategy.ts

```diff
@@ -134,11 +134,11 @@ function createState(url: string,
 
 function simulatePageNavigation(strategy: NSLocationStrategy, url: string, outletName: string) {
     strategy.pushState(null, null, url, null);
-    strategy._beginPageNavigation(outletName);
+    strategy._beginPageNavigation(outletName, null);
```

Comment:
Do we need to explicitly pass `null` here to clarify or it can be an optional param?